src/errors/__tests__/errors.test.js

```diff
@@ -0,0 +1,60 @@
+import {DownloadError, errorMessages, PatientRecordNotFoundError, PresignedUrlNotFoundError} from "../errors";
+import { AcknowledgementErrorCode } from "../../constants/enums";
+import expect from "expect";
+import { logError } from "../../middleware/logging";
+
+// mocking
+jest.mock('../../middleware/logging');
+
+describe('errors tests', () => {
+  const negativeAcknowledgementErrors = [
+    {
+      type: PatientRecordNotFoundError,
+      acknowledgementErrorCode: AcknowledgementErrorCode.ERROR_CODE_06_A,
+      errorMessage: errorMessages.PATIENT_RECORD_NOT_FOUND_ERROR
+    },
+    {
+      type: PatientRecordNotFoundError,
+      acknowledgementErrorCode: AcknowledgementErrorCode.ERROR_CODE_06_B,
+      errorMessage: errorMessages.PATIENT_RECORD_NOT_FOUND_ERROR
```

Comment:
This is the same message as `06-A`?

Reply:
Yes, the 'errorMessage' on the PatientRecordNotFoundError is the same for both the 06-A and 06-B. 

It's something I was wanting to take another look at at some point... have our internalErrorDescriptions replaced these?